torch/_higher_order_ops/cond.py

```diff
@@ -75,7 +75,7 @@ def cond(pred, true_branch, false_branch, operands):
           have consistent input and outputs, meaning the inputs have to be
           the same, and the outputs have to be the same type and shape.
 
-        operands (Tuple[torch.Tensor]): A tuple of inputs to the true/false functions.
+        operands (Tuple[Pytree[torch.Tensor]]): A tuple of inputs to the true/false functions.
```

Comment:
Ditto on the pytree feedback here: "pytree" is our terminology for "possibly nested collection of tensors", not sure if users will know what's up

Reply:
yeah, let me think about how to use a different approach of describing it.